keel-web/keel-web.gradle.kts

```diff
@@ -51,7 +51,7 @@ dependencies {
   testImplementation("com.netflix.spinnaker.kork:kork-security")
   testImplementation("com.squareup.okhttp3:mockwebserver")
   testImplementation("org.testcontainers:mysql")
-  testImplementation("org.openapi4j:openapi-schema-validator:0.7")
+  testImplementation("com.networknt:json-schema-validator:1.0.43")
```

Comment:
Validator lib for JSON schema rather than Open API. This is used in `ApiDocCompatibilityTests`